lib/src/app.dart

```diff
@@ -2,11 +2,12 @@ import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
 
+import '../../src/screens/navigation_screen.dart';
```

Comment:
can we change to this?
../src/screens/navigation_screen.dart';

Reply:
i changed it